spongehome.go

```diff
@@ -26,23 +26,68 @@
 package main
 
 import (
-    "github.com/SpongePowered/SpongeHome/controllers"
-    "github.com/go-macaron/pongo2"
-    "gopkg.in/macaron.v1"
+	"fmt"
+	"github.com/SpongePowered/SpongeHome/controllers"
+	"github.com/go-macaron/pongo2"
+	"github.com/sethvargo/go-fastly"
+	"gopkg.in/macaron.v1"
+	"os"
+	"time"
 )
 
+func clearFastly() {
+	// Clear Fastly cache
+	if os.Getenv("FASTLY_KEY") != "" {
+		time.Sleep(10 * time.Second) // Let the http server startup
+		fmt.Println("starting fastly client")
+		client, err := fastly.NewClient(os.Getenv("FASTLY_KEY"))
+		if err != nil {
+			fmt.Println(err)
```

Comment:
If `err != nil` you'll print the error, then try proceeding with what is potentially a `nil` client, which might lead to a panic.